config/database.yml

```diff
@@ -33,19 +33,19 @@ development:
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user that initialized the database.
-  #username: diaper
+  username: postgres
```

Comment:
Restore it to what it had previously -- not everyone uses `postgres`/`postgres` for their DB creds